tools/reviewer/job/sync/DiffPatch.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.job.sync;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DiffPatch {
```

Comment:
Can you describe this class?, and also `DiffHunk`? Describe what are `DiffPatch` and `DiffHunk` strings, their structure etc.

Reply:
Done.
`DiffPatch` class is renamed to `Patch`. It is more rightly to GitHub API naming.